tgui/packages/tgui/interfaces/CrewConsole.js

```diff
@@ -43,17 +44,23 @@ const jobToColor = jobId => {
   if (jobId >= 50 && jobId < 60) {
     return COLORS.department.cargo;
   }
-  if (jobId >= 200 && jobId < 230) {
+  if (jobId >= 60 && jobId < 80) {
+    return COLORS.department.service;
+  }
+  // SKYRAT EDIT - ORIGINAL: if (jobId >= 200 && jobId < 230) {
+  if (jobId >= 200 && jobId < 240) {
     return COLORS.department.centcom;
   }
   return COLORS.department.other;
 };
 
+// SKYRAT EDIT - START:
 const healthToAttribute = (oxy, tox, burn, brute, attributeList) => {
   const healthSum = oxy + tox + burn + brute;
-  const level = Math.min(Math.max(Math.ceil(healthSum / 25), 0), 5);
+  const level = Math.min(Math.max(Math.ceil(healthSum / 31), 0), 5);
```

Comment:
You should've just put the `// SKYRAT EDIT` at the end of this line instead, since it's just the change of one value.